docs/production-install.rst

```diff
@@ -5,7 +5,7 @@
 Production Install
 ==================
 
-Currently Socorro is supported on RedHat Linux 6 (and clones like CentOS)
+Currently Socorro is supported on CentOS 7
```

Comment:
Right but we haven't tested on these systems, and we are providing links like ```http://yum.postgresql.org/9.3/redhat/rhel-7-x86_64/pgdg-centos93-9.3-1.noarch.rpm``` - there are probably other minor differences, but I don't want to lead people astray. I'm sure if someone really wants to run on RH or Fedora they can figure it out without much fuss of course.